src/erasure-code/lrc/ErasureCodeLrc.h

```diff
@@ -87,6 +87,7 @@ class ErasureCodeLrc : public ErasureCode {
   std::set<int> get_erasures(const std::set<int> &need,
 			const std::set<int> &available) const;
 
+  using ErasureCode::minimum_to_decode;
```

Comment:
@adamemerson i was considering this approach before posting #18683, the upside of it is that it has less code churn. but at last i decided to add a prefix to the "internal" methods. because it helps to make the relation of these method more visible.

what do you think?

Reply:
That sounds good. I wasn't sure if the other methods are supposed to be INTERNET or if they were deprecated and simply needing replacement.

Reply:
That sounds good. I wasn't sure if the other methods were supposed to be INTERNAL or deprecated or live alongside each other or what.,

If they're internal they should DEFINITELY be private and prefixed.